30_ComputeShaderPathTracer/main.cpp

```diff
@@ -986,8 +957,8 @@ class ComputeShaderPathtracer final : public examples::SimpleWindowedApplication
 				{
 					cmdbuf->bindComputePipeline(m_PTPipeline.get());
 					cmdbuf->bindDescriptorSets(EPBP_COMPUTE, m_PTPipeline->getLayout(), 0u, 1u, &m_descriptorSet0.get());
-					cmdbuf->bindDescriptorSets(EPBP_COMPUTE, m_PTPipeline->getLayout(), 1u, 1u, &m_uboDescriptorSet1.get());
 					cmdbuf->bindDescriptorSets(EPBP_COMPUTE, m_PTPipeline->getLayout(), 2u, 1u, &m_descriptorSet2.get());
+					cmdbuf->pushConstants(m_PTPipeline->getLayout(), IShader::E_SHADER_STAGE::ESS_COMPUTE, 0, sizeof(SBasicViewParametersAligned), &pc);
```

Comment:
use some other struct, `SBasicViewParametersAligned` are too big to fix in push constants

Reply:
Done